website/src/pages/index.js

```diff
@@ -64,28 +61,22 @@ function Home() {
             <ul>
               <li>
                 {' '}
-                <Link href="/jb2/blog" variant="contained">
-                  Download latest web release
-                </Link>
+                <Link href="/jb2/blog">Download latest web release</Link>
               </li>
               <li>
-                <Link href={currentLink} variant="contained">
-                  Browse web demo instance
-                </Link>
+                <Link href={currentLink}>Browse web demo instance</Link>
               </li>
             </ul>
             <h3>Embedded</h3>
             <ul>
               <li>
-                <Link href={storybookLink} variant="contained">
+                <Link href="https://www.npmjs.com/package/@jbrowse/react-linear-genome-view">
                   Linear genome view React component on <tt>npm</tt>
-                </Link>
+                </Link>{' '}
+                <Link href={storybookLink}>(example usages)</Link>
```

Comment:
This is minor, but could we change "example usages" to "docs"? Just since there is more docs there than just examples.